### PR TITLE
[Cypress] State Machine Test | CU - 2rqec9x, 2rqecb7, 2rqecd2

### PR DESCRIPTION
**Pull request by harkiratsm**
Signed-off-by: harkiratsm <multaniharry714@gmail.com>

**Description**

This PR fixes #
- [x] DO allow user to create duplicate design (save as) if all changes are saved.
- [x] DO save design every time node attributes are configured.
- [x] DO save design with client-side validation failures.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
Task linked: [CU-2rqec9x DO save design every time node attributes are configured.](https://app.clickup.com/t/2rqec9x)

File: cypress/integration/e2e/machine_spec.js [hunk 1]
```diff
@@ -59,6 +59,41 @@ describe("State Machine Spec", () => {
...
+            cy.get("#Application_replicas").type(100);
+            // Check Saving the design
+            cy.wait("@patternSave");
+        });
+        it("DO save design with client-side validation failures",() => {
+            // Drop a node on canvas
+            cy.get(".component-drawer-svg-container[draggable='true']").first().click().drag("#cy-canvas-container", {force: true});
+            cy.wait(1000);
+            // Changing node attributes
+            cy.get("#Application_replicas").type(999999);
```
Comment (Abhishek-kumar09) [inline cypress/integration/e2e/machine_spec.js:76]: is this id written somewhere?

Comment (harkiratsm) [reply cypress/integration/e2e/machine_spec.js:76]: not exactly written but generated by the rjsf form

**Comment by leecalcote** [comment]
@harkiratsm merge conflict here.

Are we closing this PR or are you going to finish it?

**Pull request closed without merging**